dll/ntdll/ldr/ldrapi.c

```diff
@@ -1083,8 +1084,7 @@ LdrQueryProcessModuleInformationEx(
                 ModulePtr++;
 
                 /* Increase number of modules */
-                if (ModuleInformation)
-                    ModuleInformation->NumberOfModules++;
+                ModuleInformation->NumberOfModules++;
```

Comment:
This change is plain wrong.

Reply:
See above:
`ModulePtr->LoadOrderIndex = ModuleInformation->NumberOfModules;`
If `ModuleInformation` is `NULL`, an exception will occurred and never reach here, so I think `if (ModuleInformation)` is superfluous and can be removed.